uw-frame-components/portal/settings/partials/user-settings.html

```diff
@@ -9,25 +9,15 @@ <h4 class="md-subhead" ng-if="kvEnabled">Reset Zone<br>
         <small>Reset various features to their default states.</small>
       </h4>
       <div ng-if="kvEnabled" layout="column" layout-align="center start">
-        <!-- RESET ANNOUNCEMENTS -->
-        <div layout-xs="column" layout-align-xs="center start" layout="row" layout-align="start center" ng-if="FEATURES.enabled">
+        <!-- RESET MESSAGES -->
+        <div layout-xs="column" layout-align-xs="center start" layout="row" layout-align="start center">
           <md-button class="md-primary md-raised"
-                     ng-click="resetAnnouncements()"
+                     ng-click="resetMessages()"
```

Comment:
does this reset not just announcements but notifications as well? (I'm looking at the aria-label on the next line)